### ISSUE TITLE
SlimMobilenet accuracy

### ISSUE THREAD

**Issue by liyy201912** [PaddlePaddle/PaddleSlim#472]
I tried to reproduce the 80% top1 performance of SlimMobilenet(V5), but can only get around 72.3% top 1 accuracy. Would you release the pretrained model in the future?

**Comment by xiteng1988** [comment]
The 80% top1 performance is based on distillation. Even train from scratch without distillation,  72.3% is too low (it seems that your may overfit to the training data where there is a large gap between top 1 acc on training and validation). Can you reproduce the mobilenet_v3 baseline which is 75.2%?   Yes, sure. we will release the pretrained model in the future.
